less/index.less

```diff
@@ -4,56 +4,374 @@
 }
 /* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
-	display: block;
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
 }
 body {
-	line-height: 1;
+  line-height: 1;
 }
-ol, ul {
-	list-style: none;
+ol,
+ul {
+  list-style: none;
 }
-blockquote, q {
-	quotes: none;
+blockquote,
+q {
+  quotes: none;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
 }
 table {
-	border-collapse: collapse;
-	border-spacing: 0;
+  border-collapse: collapse;
+  border-spacing: 0;
 }
 
 /* Set every element's box-sizing to border-box */
 * {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+}
+
+html {
+  font-size: 62.5%;
+}
+//variables
+@color-base: #b6bde3;
+@width: 100px;
+@height: 20px;
+@link-color: black;
+@link-color-hover: purple;
+//usage
+.container {
+  width: 80%;
+  line-height: 1.5;
+  margin: 0 auto;
+  @media only screen and (max-width: 500px) {
+	background: lavender;  
+  }
+}
+header, footer {
+  display: flex;
+  justify-content: flex-end;
+  padding: 2rem;
+  nav {
+    display: flex;
+    justify-content: space-around;
+    width: 40%;
+	font-size: 14px;
+	@media only screen and (max-width: 900px){
+		display: flex;
+		flex-direction: column;
+		justify-content: flex-start;
+	}
+	@media only screen and (max-width: 500px) {
+			display: flex;
+			height: 15%;
+			width: 100%;
+			font-size: 1rem;
+			padding: 1rem;
+		}
+  }
+  a {
+    color: @link-color;
+	text-decoration: none;
+    &:hover {
+      text-decoration: underline;
+      cursor: pointer;
+	  color: @link-color-hover;
+	  @media only screen and (max-width: 500px) {
+			color: blue;
+		}
+	  
+	}
+}
+}
+.headerimg {
+  height: @height*12;
+  width: @width*4;
+  @media only screen and (max-width: 500px){
+	  align-content: center;
+	  height: 20%;
+	  width: 95%;
+  }
+  
+
+//nesting and some mixins
+}
+.content {
+  background: @color-base;
+  .intro {
+    background: #abaedb;
+  }
+  h1 {
+    font-size: 5rem;
+    border-bottom: 2px solid black;
+    display: flex;
+	justify-self: center;
+	@media only screen and (max-width: 800px){
+		font-size: 4rem;
+	}
+	@media only screen and (max-width: 500px){
+		font-size: 2rem;
+	}
+  }
+  h5 {
+    width: 55%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+	font-size: 1.2rem;
+	@media only screen and (max-width:800px){
+		width: 100%;
+		font-size: 1rem;
+	}
+	 @media only screen and (max-width: 500px){
+		  display: flex;
+		  flex-direction: column;
+		  justify-content: center;
+	 }
+  }
+  #p1 {
+    font-weight: bold;
+  }
+
+  section h2 {
+    font-size: 1.5rem;
+    font-weight: bolder;
+  }
+
+  p {
+    font-size: 1.2rem;
+  }
+  .skills, .personal-interests {
```

Comment:
considering they do the same thing, could skills and personal-interests be 1 classname?